browser-test/src/applicant/applicant_prevent_duplicate_submission.test.ts

```diff
@@ -0,0 +1,51 @@
+import {test} from '../support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  loginAsTestUser,
+  logout,
+  validateAccessibility,
+  validateScreenshot,
+  waitForPageJsLoad,
+} from '../support'
+
+test.describe('Prevent Duplicate Submission', {tag: ['@northstar']}, () => {
+  const programName = 'Test Program Name'
+
+  test.beforeEach(async ({page, adminPrograms, applicantQuestions}) => {
+    await test.step('Create program', async () => {
+      await loginAsAdmin(page)
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.publishProgram(programName)
+      await logout(page)
+    })
+
+    await test.step('As applicant, submit an application', async () => {
```

Comment:
suggestion (non-blocking): I have a slight preference for moving this `test.step` into the test below because I think the intended behavior is clearer with both steps together.